Rakefile

```diff
@@ -69,7 +69,7 @@ end
 
 desc 'Install packaging_rake_tasks gem package'
 task :install => :tarball do
-  sh 'gem install package/packaging_rake_tasks*.gem'
+  sh 'sudo gem install --local package/packaging_rake_tasks*.gem'
```

Comment:
to be honest I do not like much this change as forces sudo usage and on some machines sudo is not even available. for make install you also need to use sudo make install

Reply:
There is a difference to `make`, first you call `make` then `sudo make install` so only the `install` part runs with the `root` permissions.

If you run `sudo rake install` here it will also create the `*.gem` file as `root`. That's bad, I'd like to run only the really necessary steps as `root`, if there is a bug in creating the gem file then you could break your system easily.

Moreover if you want to delete the gem you also need `sudo`:
```console
# ls -l package/*.gem
-rw-r--r-- 1 root root 23552 Mar 24 17:52 package/packaging_rake_tasks-1.5.0.gem
```

I'dd add `root` and `sudo` detection....